pkgs/os-specific/linux/tomb/default.nix

```diff
@@ -28,8 +28,6 @@ stdenv.mkDerivation rec {
     install -Dm755 tomb       $out/bin/tomb
     install -Dm644 doc/tomb.1 $out/share/man/man1/tomb.1
 
-    ln -s ${gnupg}/bin/gpg $out/bin/gpg
```

Comment:
Should gnupg be added to the wrapper below?

Reply:
Hum, I'm not sure...
If it is then it's missing [some others](https://github.com/dyne/Tomb/blob/59408b373f14a56d3a8a6dca9fc44e5ddb5f2dad/tomb#L842) like `sudo` and `mkfs`.

Reply:
We cannot add `sudo` it must be provided by the system due to its setuid bit.
On nixos it is in `/run/wrappers/bin/sudo` most other linux distributions have it in `/usr/bin/sudo`. Adding `mkfs` could be done.

Reply:
It is done.